src/ApiCore/Testing/MockStubTrait.php

```diff
@@ -268,7 +270,8 @@ public static function create($responseObject, $status = null, $deserialize = nu
      * Creates a sequence such that the responses are returned in order.
      * @param mixed[] $sequence
      * @param callable $deserialize
-     * @return MockTransport
+     * @param Status $finalStatus
+     * @return MockStubTrait
```

Comment:
I'm not a big fan of this. Traits cannot be used as type hints, the type returned will not be `MockStubTrait`, and `$res instanceof MockStubTrait` will return false.

Could we say `@return static An instance of the current class type`, or something simiar?